pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -397,6 +442,12 @@ func ParseTunnelInterfaceOptions(portData *OVSPortData) (net.IP, string) {
 	return remoteIP, psk
 }
 
+// CreateUplinkPort creates uplink port.
+func (br *OVSBridge) CreateUplinkPort(name string, ifDev string, ofPortRequest int32, externalIDs map[string]interface{}) (string, Error) {
+	br.uplinkName = ifDev
```

Comment:
I still feel no need to save uplink name in Bridge. It comes from the caller (agentInitiator) and consumed by the caller, and you could just save the uplink name there, rather than in the Bridge.
Again if I remember it correctly we now do not save any attributes from the callers in Bridge, except bridge name.

Reply:
Correct. I agree to save the uplink name in caller. Moved "uplinkName" in nodeConfig in update.